mesheryctl/internal/cli/root/pattern/view_test.go

```diff
@@ -46,6 +47,24 @@ func TestPatternView(t *testing.T) {
 			Token:            filepath.Join(fixturesDir, "token.golden"),
 			ExpectError:      false,
 		},
+		{
+			Name:             "View Specific Pattern by Name",
+			Args:             []string{"view", "design"},
+			ExpectedResponse: "view.pattern.output.golden",
+			Fixture:          "view.pattern.api.response.golden",
+			URL:              testContext.BaseURL + "/api/pattern?name=Untitled Design",
```

Comment:
Currently a design with this name exists. But if gets deleted in the future, this test might fail. We could add a block before this to upload a design with a name, and then use that name in this block. This will ensure that the file always exists when the test runs...